### DISCUSSION TITLE
Add white boarder around icons

### DISCUSSION THREAD

**Discussion by DeNohvo** [Jyers/Marlin#65] [Ideas]
In the Smith3D firmware they added a white boarder around the selected menu item, this will help when the screen is viewed from an angle. Hopefully this does not mean the screen needs to be flashed.

https://i1.wp.com/smith3d.com/wp-content/uploads/2020/07/IMG_20200710_201755.jpg?w=900&ssl=1

**Comment by Jyers** [comment]
As far as I can tell this is already the case on this firmware. Is there something I am missing here?

**Comment by Schnabulation** [comment]
It‘s already in the standard 2.0.x-bugfix branch of Marlin - and also enabled on this fork. Just tested it...

**Reply by DeNohvo** [reply]
Yep
